src/backend/db/videos.js

```diff
@@ -3,19 +3,88 @@
  * You can add videos of your wish with different attributes
  * */
 
+ import { v4 as uuid } from "uuid";
+
 export const videos = [
   {
-    _id: "Wo5dMEP_BbI",
+    _id: uuid(),
```

Comment:
dont add id by calling this function will cause error in the future, contact me for this if need explanation